### ISSUE TITLE
Setup_ve failed

### ISSUE THREAD

**Issue by donbalon4** [Shippable/support#3383]
Hi, I'm getting this ![imagen](https://cloud.githubusercontent.com/assets/3063810/23172651/3a871aa4-f835-11e6-8132-305922395031.png)
 I have no changed my shippable.yml file

**Comment by kanthchandra** [comment]
Hi @donbalon4 Can you please point us to the build that you are referring here?

**Comment by donbalon4** [comment]
@kanthchandra , this [link](https://app.shippable.com/runs/58ac60d0500f1b1000f98465/1/console) is useful?

**Comment by stephanielingwood** [comment]
Hi @donbalon4,

Sorry you're running into this. Yes, we recently made changes to how we handle errors given by virtualenv, in response to [this customer request](https://github.com/Shippable/support/issues/3289). 

For python, we only support major/minor versions in the yml - for example, 2.7 or 3.5. If you use 2.7.6, virtualenv gives an error. Before the recent release, we would ignore those virtualenv errors and continue the build if we could use the default version of python - which happens to be 2.7.6. So, users like you had successful builds by specifying `2.7.6`, but not for the reasons you might have thought.

To give better visibility into your build, we decided to stop ignoring virtualenv errors -.which means that yes, your build will now fail with `2.7.6` in he yml.  Luckily, it's an easy fix: change your yml to say `2.7` instead of `2.7.6`, and your builds will start succeeding again. 

Hope this helps. Sorry for any inconvenience.
